src/main/java/io/cryostat/net/web/http/api/v1/TargetRecordingDeleteHandler.java

```diff
@@ -103,6 +110,13 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
                     if (descriptor.isPresent()) {
                         connection.getService().close(descriptor.get());
                         reportService.delete(connectionDescriptor, recordingName);
+                        notificationFactory
+                                .createBuilder()
+                                .metaCategory(NOTIFICATION_CATEGORY)
+                                .metaType(HttpMimeType.JSON)
+                                .message(Map.of("recording", recordingName))
```

Comment:
Maybe this payload should also include the `targetId` (really a JMX service URL) of the target that "owns" the named recording?

Reply:
Makes sense, I've added this to the target-oriented handlers' payloads